github/orgs_packages_test.go

```diff
@@ -0,0 +1,368 @@
+			  "subscriptions_url": "https://api.github.com/users/github/subscriptions",
+			  "organizations_url": "https://api.github.com/users/github/orgs",
+			  "repos_url": "https://api.github.com/users/github/repos",
+			  "events_url": "https://api.github.com/users/github/events{/privacy}",
+			  "received_events_url": "https://api.github.com/users/github/received_events",
+			  "type": "Organization",
+			  "site_admin": false
+			},
+			"version_count": 1,
+			"visibility": "private",
+			"url": "https://api.github.com/orgs/github/packages/container/hello_docker",
+			"created_at": `+referenceTimeStr+`,
+			"updated_at": `+referenceTimeStr+`,
+			"html_url": "https://github.com/orgs/github/packages/container/package/hello_docker"
+		  }
+		  ]`)
+	})
+
+	ctx := context.Background()
+	packages, _, err := client.Organizations.ListPackages(ctx, "o", nil)
+	if err != nil {
+		t.Errorf("Organizations.ListPackages returned error: %v", err)
+	}
+
+	want := []*Package{{
+		ID:           Int64(197),
+		Name:         String("hello_docker"),
+		PackageType:  String("container"),
+		VersionCount: Int64(1),
+		Visibility:   String("private"),
+		URL:          String("https://api.github.com/orgs/github/packages/container/hello_docker"),
+		HTMLURL:      String("https://github.com/orgs/github/packages/container/package/hello_docker"),
+		CreatedAt:    &Timestamp{referenceTime},
+		UpdatedAt:    &Timestamp{referenceTime},
+		Owner: &User{
+			Login:             String("github"),
+			ID:                Int64(9919),
+			NodeID:            String("MDEyOk9yZ2FuaXphdGlvbjk5MTk="),
+			AvatarURL:         String("https://avatars.githubusercontent.com/u/9919?v=4"),
+			GravatarID:        String(""),
+			URL:               String("https://api.github.com/users/github"),
+			HTMLURL:           String("https://github.com/github"),
+			FollowersURL:      String("https://api.github.com/users/github/followers"),
+			FollowingURL:      String("https://api.github.com/users/github/following{/other_user}"),
+			GistsURL:          String("https://api.github.com/users/github/gists{/gist_id}"),
+			StarredURL:        String("https://api.github.com/users/github/starred{/owner}{/repo}"),
+			SubscriptionsURL:  String("https://api.github.com/users/github/subscriptions"),
+			OrganizationsURL:  String("https://api.github.com/users/github/orgs"),
+			ReposURL:          String("https://api.github.com/users/github/repos"),
+			EventsURL:         String("https://api.github.com/users/github/events{/privacy}"),
+			ReceivedEventsURL: String("https://api.github.com/users/github/received_events"),
+			Type:              String("Organization"),
+			SiteAdmin:         Bool(false),
+		},
+	}}
+	if !cmp.Equal(packages, want) {
+		t.Errorf("Organizations.ListPackages returned %+v, want %+v", packages, want)
+	}
+
+	const methodName = "ListPackages"
+	testNewRequestAndDoFailure(t, methodName, client, func() (*Response, error) {
```

Comment:
For every unit test that you added, can you please follow the pattern of other tests in this repo and also call 

```
testBadOptions(t, methodName, func() (err error)
```

?  (Please see `repos_test.go` for many examples.)

This will help with the code coverage.  Thank you!